<launch>
    
    <arg name="rviz" default="true" />

    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" required="true" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="camera_info_url" value="file://$(find bender_perception)/config/usb_camera_info.yaml" />
        <rosparam command="load" file="$(find bender_perception)/config/usb_camera_params.yaml" />
        <remap from="/usb_cam/image_raw" to="/bender_camera/image_raw" />
    </node>

    <node name="$(anon rviz)" if="$(arg rviz)" pkg="rviz" type="rviz" 
        respawn="false" args="-d $(find bender_perception)/config/debug.rviz" output="screen"/>

    <include file="$(find bender_perception)/launch/vision.launch" />

</launch>